src/api/db/migrate/20170607105625_use_rails_enums_binary_releases_table.rb

```diff
@@ -0,0 +1,37 @@
+class UseRailsEnumsBinaryReleasesTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table(:binary_releases) do |t|
+      t.column :new_operation, :integer, limit: 2, default: 0
+    end
+
+    BinaryRelease.operations.values.each do |index|
+      # We need to use raw sql to avoid conflicts with the ActiveRecord::Enum
+      # mapping defined in the model.
+      # SQL enums start at 1, ActiveRecord::Enum usually at 0.
+      BinaryRelease.connection.
+        execute("UPDATE binary_releases SET new_operation='#{index}' WHERE operation='#{BinaryRelease.operations.key(index)}'")
+    end
+
+    remove_column :binary_releases, :operation
+    rename_column :binary_releases, :new_operation, :operation
+  end
```

Comment:
I will. I was even considering to keep the old column around and drop them later, when we are sure everything still works.